HTMessaging/Classes/Connection.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  Connection.swift
+//  HTMessaging
+//
+//  Created by Hector Jose Alvarado Chapa on 22/12/19.
+//  Copyright © 2019 Hector Jose Alvarado Chapa. All rights reserved.
+//
+
+import Foundation
+import Firebase
+
+public final class Connection {
+    func withFirebaseFile(with fileName: String, ofType: Filetype, completion: @escaping(_ status: FIRResponse?) -> Void) {
```

Comment:
Please add a white line every time  a class opens a bracket and before it finishes too, like this:

class T {

    func t()

}

instead of :

class T {
    func t()
}